examples/website-demos/social-messaging/src/components/ChannelContainer/ChannelContainer.tsx

```diff
@@ -81,7 +82,7 @@ export const ChannelContainer: React.FC = () => {
         {isNewChat ? <NewChat /> : <MessageList />}
         <MessageInput mentionAllAppUsers />
       </Window>
-      {/* <Thread /> */}
+      <SocialThreadInner />
```

Comment:
let's just call this SocialThread, an Inner component would be one nested as a child within the outer layer